src/components/PaymentRequestButtonElement.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import React from 'react';
+import PropTypes from 'prop-types';
+import shallowEqual from '../utils/shallowEqual';
+import type {ElementContext} from './Elements';
+
+type Props = {
+  className: string,
+  elementRef: Function,
+  onBlur: Function,
+  onFocus: Function,
+  onReady: Function,
+  paymentRequest: {
+    canMakePayment: Function,
+    on: Function,
+    show: Function,
+  },
+};
+
+const noop = () => {};
```

Comment:
For example, if we were using `SyntheticEvent<T>` for our props, I imagine Flow would catch that `noop` should take a single `event` argument, instead of "no" arguments.